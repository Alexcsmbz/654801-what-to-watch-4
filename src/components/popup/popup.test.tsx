import Popup from './popup';

describe(`Popup snapshot test`, () => {
  it(`Popup should render popup`, () => {
    const tree = renderer.create(
        <Popup message={`message`} />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
