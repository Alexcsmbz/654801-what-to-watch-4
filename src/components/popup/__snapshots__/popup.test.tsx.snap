// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Popup snapshot test Popup should render popup 1`] = `
Array [
  <div
    className="sc-AxiKw eWEzSA"
  />,
  <div
    className="sc-AxjAm freQmy"
  >
    <span>
      message
    </span>
    <div
      className="sc-AxirZ jpowrm"
    >
      ðŸ’©
    </div>
    <button
      className="btn"
      onClick={[Function]}
      type="button"
    >
      <span>
        Reload app
      </span>
    </button>
  </div>,
]
`;
